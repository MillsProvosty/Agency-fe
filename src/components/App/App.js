import React from "react";
import "./App.css";
import { FaHandsHelping } from "react-icons/fa";

function App() {
  return (
    <section className="App">
      <FaHandsHelping />
      <p>Agency is pretty frickin great</p>
      <section>
        <button>Volunteer</button>
        <button>Client</button>
      </section>
    </section>
  );
}

export default App;
