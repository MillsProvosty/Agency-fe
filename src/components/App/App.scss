* {
  box-sizing: border-box;
}

.App {
  text-align: center;
  background-color: aliceblue;
  font-family: 'Nunito Sans', sans-serif;
  color: #37474E;


  & section {
    justify-content: space-around;
  }
}
