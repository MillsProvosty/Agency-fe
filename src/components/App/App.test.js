import React from 'react';
import { shallow } from 'enzyme';
import { App } from './App';


describe('App', () => {
  let wrapper; 

  const setState = jest.fn();
  const useStateSpy = jest.spyOn(React, 'useState')
  useStateSpy.mockImplementation((init) => [init, setState]);
  
  beforeEach(() => {
    wrapper = shallow(
      <App />
    )
  });
  
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot() 
  });

})
