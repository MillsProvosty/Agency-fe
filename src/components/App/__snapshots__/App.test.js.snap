// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<section
  className="App"
>
  <Route
    path="/profile"
    render={[Function]}
  />
  <Route
    path="/user-form"
    render={[Function]}
  />
  <Route
    path="/"
    render={[Function]}
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Schedule)",
        "type": [Function],
      }
    }
    exact={true}
    path="/schedule"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/team"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(Profile)",
        "type": [Function],
      }
    }
    exact={true}
    path="/profile"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/about"
  />
</section>
`;
